src/DataViews/DataView.cpp

```diff
@@ -365,4 +369,14 @@ void DataView::OnCopySelectionRequested(const std::vector<int>& selection) {
   app_->SetClipboard(clipboard);
 }
 
+uint64_t DataView::GetScopeId(uint32_t row) const {
```

Comment:
So do we have DataViews, where the elements are not scopes? (Thinking of sampling tab).